src/ncp/ncp_base.cpp

```diff
@@ -46,6 +46,7 @@
 
 #include "common/code_utils.hpp"
 #include "common/debug.hpp"
+#include "mac/sub_mac.hpp"
```

Comment:
Do we need this header?
Can we remove this?

We generally try to not include core-internal headers in `cli` or `ncp` module (they can of course use public OT `.h` headers and APIs).

Reply:
Got it. Also removed the header, it is not needed anymore.